apps/wallet/src/ui/app/components/ledger/ConnectLedgerModal.tsx

```diff
@@ -10,14 +10,12 @@ import { ModalDialog } from '_src/ui/app/shared/ModalDialog';
 import { Text } from '_src/ui/app/shared/text';
 
 type ConnectLedgerModalProps = {
```

Comment:
The modal is now accessible via a nested route, so there's no need to control the modal's visibility with a prop